template/coin.template

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	_ "net/http/pprof"
+	"github.com/skycoin/skycoin/src/skycoin"
+	"github.com/skycoin/skycoin/src/util/logging"
+)
+
+var (
+	// Version of the node. Can be set by -ldflags
+	Version = "{{.Version}}"
+	// Commit ID. Can be set by -ldflags
+	Commit = "{{.Commit}}"
+	// Branch name. Can be set by -ldflags
+	Branch = "{{.Branch}}"
+	// ConfigMode (possible values are "", "STANDALONE_CLIENT").
+	// This is used to change the default configuration.
+	// Can be set by -ldflags
+	ConfigMode = ""
+
+	logger = logging.MustGetLogger("main")
+
+	// GenesisSignatureStr hex string of genesis signature
+	GenesisSignatureStr = "{{.GenesisSignatureStr}}"
+	// GenesisAddressStr genesis address string
+	GenesisAddressStr = "{{.GenesisAddressStr}}"
+	// BlockchainPubkeyStr pubic key string
+	BlockchainPubkeyStr = "{{.BlockchainPubkeyStr}}"
+	// BlockchainSeckeyStr empty private key string
+	BlockchainSeckeyStr = "{{.BlockchainSeckeyStr}}"
+
+	// GenesisTimestamp genesis block create unix time
+	GenesisTimestamp uint64 = {{.GenesisTimestamp}}
+	// GenesisCoinVolume represents the coin capacity
+	GenesisCoinVolume uint64 = {{.GenesisCoinVolume}}
+
+	// DefaultConnections the default trust node addresses
+	DefaultConnections = []string{
+    {{- range $index, $address := .DefaultConnections}}
+        "{{$address -}}",
+    {{- end}}
+	}
+)
+
+var devConfig = skycoin.NodeConfig{
+	GenesisSignatureStr: GenesisSignatureStr,
+	GenesisAddressStr:   GenesisAddressStr,
+	GenesisCoinVolume:   GenesisCoinVolume,
+	GenesisTimestamp:    GenesisTimestamp,
+	BlockchainPubkeyStr: BlockchainPubkeyStr,
+	BlockchainSeckeyStr: BlockchainSeckeyStr,
+	DefaultConnections:  DefaultConnections,
+	// Disable peer exchange
+	DisablePEX: {{.DisablePEX}},
```

Comment:
Most of these options aren't meant to be configurable by the template.

They have the same defaults for all fiber coins because they control the basic app behavior.

The template needs to parse the fiber parameters that matter, merge them into a struct set with defaults then write the full struct out